### DISCUSSION TITLE
Do you want to consider adding a progress bar?

### DISCUSSION THREAD

**Discussion by wujianming** [Macad3D/Macad3D#51] [Ideas]
opening large files is particularly slow. Is there a good way to do it? It's really not good. Do you want to consider adding a progress bar?

**Comment by dipts** [comment]
Are you talking about large Macad (.model) files or large imported files? If the latter, what file format are you using for the import?

**Comment by wujianming** [comment]
step...

**Comment by wujianming** [comment]
![11](https://user-images.githubusercontent.com/3305805/226081406-00b0dd5f-e470-40ea-b7bc-b2eaa0165836.png)
This effect occurs when importing an 11.7M step file. The export time generally takes more than 1 minute
